time_tracker_api/time_entries/time_entries_dao.py

```diff
@@ -226,6 +226,9 @@ def create(self, data: dict):
     def update(self, id, data: dict, description=None):
         event_ctx = self.create_event_context("update", description)
 
+        if data.get('update_last_entry_if_overlap'):
```

Comment:
I think, this get, should have a False default value, to make it more robust.

Reply:
Also, this code should be after the line `self.check_whether_current_user_owns_item(time_entry)`